packages/app-desktop/utils/window/eventHandlerOverrides.test.ts

```
@@ -0,0 +1,84 @@
+		const originalAddEventListener = document.addEventListener;
+		document.addEventListener = jest.fn((event, handler) => {
+			if (event === 'click') {
+				clickHandler = handler as (event: unknown)=> void;
+			}
+		}) as unknown as typeof document.addEventListener;
+
+		// Load and execute the script
+		const scriptPath = path.join(__dirname, 'eventHandlerOverrides.js');
+		const scriptContent = fs.readFileSync(scriptPath, 'utf8');
+		// Run in global scope
+		eval(scriptContent);
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)